parsons/utilities/format_phone_number.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+
+def format_phone_number(phone_number, country_code="1"):
+    """
+    Formats a phone number in E.164 format.
```

Comment:
Could you add some documentation showing what it'll do? That would be helpful to someone who doesn't know what the e.164 format is, and might have no idea what this does.

Even one line like:

`Example: Converts "(123) 123-1212" to "+01231231212"`

would be very helpful!

Reply:
Sounds good. Went ahead and made that change.